cassovary-core/src/main/scala/com/twitter/cassovary/util/SortedArrayOps.scala

```diff
@@ -0,0 +1,85 @@
+package com.twitter.cassovary.util
+
+import java.util
+
+import scala.collection.mutable
+import scala.reflect.ClassTag
+
+object SortedArrayOps {
+  /**
+   * Finds element in the SortedSeq using provided ordering.
+   *
+   * Assumes that the collection is sorted using this ordering.
+   */
+  def findUsingOrdering(array: Array[Int], t: Int): Boolean = {
+    util.Arrays.binarySearch(array, t) match {
```

Comment:
that's right. Please add a comment then that we are writing our own to not have boxing/unboxing overhead of scala.collection.Searching